sdk/ai/Azure.AI.Projects/tests/Samples/Inference/Sample_Embeddings.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+using Azure.AI.Inference;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.AI.Projects.Tests;
+
+public class Sample_Embeddings : SamplesBase<AIProjectsTestEnvironment>
+{
+    [Test]
+    public void BasicEmbedding()
+    {
+        var connectionString = TestEnvironment.AzureAICONNECTIONSTRING;
+        EmbeddingsClient embeddingsClient = new AIProjectClient(connectionString, new DefaultAzureCredential()).GetEmbeddingsClient();
+
+        var input = new List<string> { "King", "Queen", "Jack", "Page" };
+        var requestOptions = new EmbeddingsOptions(input);
+
+        Response<EmbeddingsResult> response = embeddingsClient.Embed(requestOptions);
```

Comment:
You need to specify the model name

Reply:
Updated!

I was just following the inference samples.